services/syllabus/decorators.py

```diff
@@ -0,0 +1,138 @@
+##########################################################################
+# Name:     validate
+# Purpose: File contains all the decorators to validate the user request
+#
+# Author:     Sai Nikhita Guduur
+#
+# Created:   10/07/2019
+# Copyright:   (c) Hoder Solutions Pvt Ltd 2018 - Present
+# Licence:   <your licence>
+##########################################################################
+from flask import request, Response
+from json import dumps
+from functools import wraps
+from services.syllabus.models import Syllabus
+
+
+def validate_add_syllabus(func):
```

Comment:
if 'id' is passed from the caller, we need very minimal validations in this file

Reply:
contact Kalyan regarding this point